src/shipLogEntries.ts

```diff
@@ -14,31 +14,14 @@ import type { StructuredFields } from './model';
 import { getStructuredFields } from './structuredFields';
 
 const { awsConfig } = getCommonConfig();
-const {
-	kinesisStreamName,
-	kinesisStreamRole,
-	structuredDataBucket,
-	structuredDataKey,
-} = getShipLogsConfig();
+const { kinesisStreamName, structuredDataBucket, structuredDataKey } =
+	getShipLogsConfig();
 
 const s3 = new S3(awsConfig);
-const kinesis = getKinesisClient(awsConfig, kinesisStreamRole);
+const kinesis = getKinesisClient(awsConfig);
 
-function getKinesisClient(
-	awsConfig: ConfigurationOptions,
-	role: string | undefined,
-): Kinesis {
-	if (role) {
-		const credentials = new ChainableTemporaryCredentials({
-			params: {
-				RoleArn: role,
-				RoleSessionName: `shipLogEntries-lambda`,
-			},
-		});
-		return new Kinesis({ ...awsConfig, credentials });
-	} else {
-		return new Kinesis(awsConfig);
-	}
+function getKinesisClient(awsConfig: ConfigurationOptions): Kinesis {
```

Comment:
Is it fair to say this abstraction isn't doing much work now, and can be removed?

Reply:
Ah yes, good spot! Updated now.